core/templates/pages/android-page/android-page.component.ts

```diff
@@ -135,6 +143,22 @@ export class AndroidPageComponent implements OnInit, OnDestroy {
     this.featuresShown = featureNumber;
   }
 
+  validateEmailAddress(): boolean {
+    // A simple check for basic email validation.
+    let regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
Please put the email regex to _constants.ts_

Reply:
Done.